package.json

```diff
@@ -28,6 +28,7 @@
     "stage:personal": "ELEVENTY_ENV=dev npm run build && gcloud app deploy --project web-dev-staging --quiet --no-promote",
     "deploy": "ELEVENTY_ENV=prod npm run build && node index-algolia.js && gcloud app deploy --project web-dev-production-1 --quiet",
     "snapshots": "percy exec -- node tools/percy/snapshots.js",
+    "dev:snapshots": "node tools/percy/snapshots-dev.js",
```

Comment:
I think there might be a couple of issues here. You may want this command to also build the site in test mode (which ensures that the blog always has the same 4 posts on it) and start the local server. Rather than have a separate script (snapshots-dev) could the npm script be written to look like `test:snapshots` but have it set both environment variables, ELEVENTY_ENV=test and PERCY_ENV=dev?

Reply:
So since this is literally for development to _see_ what is happening 👀, it actually doesn't matter that it has the same 4 posts on it.

I have a thought. Instead of extra env vars, could we overload `ELEVENTY_ENV=dev` to mean, show the display?

Reply:
> So since this is literally for development to see what is happening 👀, it actually doesn't matter that it has the same 4 posts on it.

I could be wrong but I think percy will still try to upload the screenshots won't it?

> I have a thought. Instead of extra env vars, could we overload ELEVENTY_ENV=dev to mean, show the display?

We can but then we wouldn't get the test posts. I guess I'm ok with us trying it and if it's annoying we'll think of another solution.